flask_resty/authentication.py

```diff
@@ -2,17 +2,13 @@
 
 # -----------------------------------------------------------------------------
 
-REQUEST_CREDENTIALS_KEY = 'request_credentials'
```

Comment:
why remove this?

Reply:
idk, do you think having this random constant is that useful? i was looking at it more and thought it added more noise than anything else